clothing_boutique/demo/website_page_views.xml

```diff
@@ -372,7 +372,7 @@ to Saturday&amp;nbsp;at 10 Am - 7:30 PM</font>
                                     <div class="col-lg-3 offset-lg-1 o_colored_level" style="">
                                         <p class="text-muted"><b>The Glam Boutique</b><br/>250 Executive Park Blvd, Suite 3400 <br/> San Francisco CA 94134 <br/>United States</p>
                                         <p class="text-muted">
-                                            <img src="/clothing_boutique/static/src/binary/ir_attachment/940-glamlogo.png" alt="" class="img img-fluid o_we_custom_image me-auto float-start" data-original-id="939" data-original-src="/web/image/939-36b131e2/glam%20logo.png" data-mimetype="image/png" data-bs-original-title="" aria-describedby="tooltip126360" style="width: 100% !important;" loading="lazy"/>
+                                            <img src="/clothing_boutique/static/src/binary/ir_attachment/939-glamlogo.png" alt="" class="img img-fluid o_we_custom_image me-auto float-start" data-original-id="939" data-original-src="/web/image/939-36b131e2/glam%20logo.png" data-mimetype="image/png" data-bs-original-title="" aria-describedby="tooltip126360" style="width: 100% !important;" loading="lazy"/>
```

Comment:
Why this change?

Reply:
File 939 does not exist, and since data-original-id="939", I guessed it was a typo error.